### PR TITLE
Fix: Tabs a11y is not performing as expected [ED-5409]

### PR DESCRIPTION
**Pull request by iNewLegend**
https://user-images.githubusercontent.com/10234691/149666619-26253c92-6f28-404e-9608-69781d043035.mov

Updated:

https://user-images.githubusercontent.com/10234691/151118316-f27a5054-10ff-468d-80b1-9267aafbd83d.mov

### CODE REVIEW

File: assets/dev/js/frontend/handlers/base-tabs.js [hunk 1]
```diff
@@ -163,6 +163,7 @@ export default class baseTabs extends elementorModules.frontend.handlers.Base {
 						break;
 					case 'Enter':
 					case 'Space':
+					case ' ': // Space.
```
Comment (Nevoss) [inline assets/dev/js/frontend/handlers/base-tabs.js:166]: Why is it here?

File: assets/dev/scss/frontend/widgets/tabs.scss [hunk 2]
```diff
@@ -61,6 +61,10 @@
 					left: 100%;
 				}
 			}
+
+			&:focus {
+				text-decoration: underline;
+			}
```
Comment (Nevoss) [inline assets/dev/scss/frontend/widgets/tabs.scss:67]: Do you say that the focus was not broken? it just was without UI? if yes I am not sure this is the solution.

**Review by Nevoss — CHANGES_REQUESTED** [review]

File: assets/dev/scss/frontend/widgets/tabs.scss (hunk 2 above)
Comment (iNewLegend) [reply assets/dev/scss/frontend/widgets/tabs.scss:67]: The answer is yes, @shilo-ey told its fine, what you suggest?

File: assets/dev/js/frontend/handlers/base-tabs.js (hunk 1 above)
Comment (iNewLegend) [reply assets/dev/js/frontend/handlers/base-tabs.js:166]: I don't know what does the `'Space'` below but when i debugged it, the event is `' '`.

**Review by Nevoss — APPROVED** [review]

**Review by nisan250 — APPROVED** [review]

File: assets/dev/js/frontend/handlers/base-tabs.js [hunk 3]
```diff
@@ -162,7 +162,6 @@ export default class baseTabs extends elementorModules.frontend.handlers.Base {
 						this.handleKeyboardNavigation( event );
 						break;
 					case 'Enter':
-					case 'Space':
```
Comment (matipojo) [inline assets/dev/js/frontend/handlers/base-tabs.js:165]: Why? `Space` should act like `Enter.
For example: 
https://www.w3.org/WAI/WCAG21/Techniques/client-side-script/SCR35#:~:text=but%20it%20also%20occurs%20when%20the%20user%20focuses%20the%20element%20and%20hits%20enter%20or%20space%2C

**Review by matipojo — CHANGES_REQUESTED** [review]

File: assets/dev/js/frontend/handlers/base-tabs.js (hunk 3 above)
Comment (iNewLegend) [reply assets/dev/js/frontend/handlers/base-tabs.js:165]: Product request, look at ED.

File: assets/dev/js/frontend/handlers/base-tabs.js [hunk 4]
```diff
@@ -162,7 +162,7 @@ export default class baseTabs extends elementorModules.frontend.handlers.Base {
...
+					case ' ': // Space.
```
Comment (matipojo) [inline assets/dev/js/frontend/handlers/base-tabs.js:165]: The `event.code` is `Space`, switch to check the `code` instead of the `key`

**Review by matipojo — CHANGES_REQUESTED** [review]

File: assets/dev/js/frontend/handlers/base-tabs.js (hunk 4 above)
Comment (iNewLegend) [reply assets/dev/js/frontend/handlers/base-tabs.js]: Handled.

**Review by matipojo — COMMENTED** [review]
Please re-open it

**Pull request merged**